### PR TITLE
feat(sdk-core): add fetchCrossChainUTXOs in wallet

### PR DESCRIPTION
**Pull request by jicli**
Ticket: BG-57867

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code compiles correctly for both Node and Browser environments
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My commits follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) and I have properly described any BREAKING CHANGES
- [x] The ticket or github issue was included in the commit message as a reference
- [x] I have made corresponding changes to the documentation and on any new/updated functions and/or methods - [jsdoc](https://jsdoc.app/)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by CapnMigraine — COMMENTED** [review]
lgtm

File: modules/sdk-core/src/bitgo/wallet/wallet.ts [hunk 1]
```diff
@@ -2087,6 +2088,16 @@ export class Wallet implements IWallet {
     return this.bitgo.del(this.url()).result();
   }
 
+  /**
+   * Fetches crossChain UTXOs
+   * Currently only for AVAX
+   * @param {string} params.sourceChain the sourcechain to pick UTXOs, if not given, then pick from all available chains [P, C]
+   */
+  async fetchCrossChainUTXOs(params: FetchCrossChainUTXOsOptions): Promise<any> {
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:2096]: Can we get the response type for `any` here?

Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:2096]:
```suggestion
 fetchCrossChainUTXOs(params: FetchCrossChainUTXOsOptions): Promise<any> {
```

File: modules/sdk-core/src/bitgo/wallet/iWallet.ts [hunk 2]
```diff
@@ -443,6 +443,10 @@ export interface SendManyOptions extends PrebuildAndSignTransactionOptions {
   gasLimit?: number;
 }
 
+export interface FetchCrossChainUTXOsOptions {
+  sourceChain?: string;
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/iWallet.ts:447]: Are `P` and `C` the only options for this param? If so, could the type be `sourceChain?: 'P' | 'C';`?

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]

File: modules/sdk-core/src/bitgo/wallet/iWallet.ts (hunk 2 above)
Comment (jicli) [reply modules/sdk-core/src/bitgo/wallet/iWallet.ts:447]: yes

**Review by CapnMigraine — APPROVED** [review]

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**